tests/build_linux.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/sh
+# shellcheck disable=SC2103
+
+export CC=/usr/bin/gcc-10 &&
```

Comment:
Wasn't this supposed to be gcc-11?

Reply:
In the branch there's currently nothing that requires GCC 11, but there will be if stringToInteger and stringToDouble are merged in. I will change it now.